src/DynamoCoreWpf/DynamoCoreWpf.csproj

```diff
@@ -4,15 +4,20 @@
   </ImportGroup>
   <PropertyGroup>
     <ProjectGuid>{51BB6014-43F7-4F31-B8D3-E3C37EBEDAF4}</ProjectGuid>
-    <OutputType>Library</OutputType>
+    <OutputType>Library</OutputType> 
     <AppDesignerFolder>Properties</AppDesignerFolder>
     <RootNamespace>Dynamo.Wpf</RootNamespace>
     <AssemblyName>DynamoCoreWpf</AssemblyName>
     <DocumentationFile>$(OutputPath)\DynamoCoreWpf.XML</DocumentationFile>
     <EnableDefaultCompileItems>False</EnableDefaultCompileItems>
     <EnableDefaultEmbeddedResourceItems>False</EnableDefaultEmbeddedResourceItems>
-    <PreferredToolArchitecture>x64</PreferredToolArchitecture>
+    <PreferredToolArchitecture>x64</PreferredToolArchitecture> 
   </PropertyGroup>
+    <PropertyGroup Condition=" '$(TargetFramework)' == 'net6.0' ">
+        <TargetFramework>net6.0-windows</TargetFramework>
+        <UseWPF>true</UseWPF>
+        <EnableDefaultPageItems>false</EnableDefaultPageItems>
```

Comment:
what does this do?

Reply:
would it be a bit more idiomatic to just remove the page item references below instead?

Reply:
when I remove page references I get a lot of errors. e.g.
I tried removing  'Controls\ ClassInformationView.xaml from pages references and I got this:
<img width="704" alt="Screenshot 2023-06-21 112603" src="https://github.com/DynamoDS/Dynamo/assets/134449953/94cd3334-c4f9-4def-8386-83c8ead51732">
It's worth notiing that they are duplicated pages, the duplicity that it is encountering are with these two types or references:
<img width="359" alt="Screenshot 2023-06-21 113155" src="https://github.com/DynamoDS/Dynamo/assets/134449953/4619d8da-c6e5-4695-92a6-b8b344a6f6d5">
AND
<img width="357" alt="Untitled" src="https://github.com/DynamoDS/Dynamo/assets/134449953/9824ee6e-7086-4791-83f1-347f37738a27">